packages/@okta/vuepress-site/guides/sign-into-spa/section03/react/attachcomponentsetc.md

```diff
@@ -1,18 +1,112 @@
 ## Attach Components to the Secure Router
 
-You need to provide these routes in your sample application so that we can sign the user in and handle the callback from Okta. We show you how to set these up below using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):
+We provide the `Security` component which makes an Auth object available to child components and the `withAuth` HOC. We show you how to set these up below using [React Router DOM](https://github.com/ReactTraining/react-router/tree/master/packages/react-router-dom):
 
-- `/`: A default home page to handle basic control of the app.
-- `/implicit/callback`: This is where auth is handled for you after redirection.
+```javascript
+import React, { Component } from 'react';
+import { BrowserRouter as Router, Route } from 'react-router-dom';
+// import config from ...
+
+class App extends Component {
+  render() {
+    return (
+      <Router>
+        <Security {...config} >
+          { /* child components have access to auth context */ }
+        </Security>
+      </Router>
+    );
+  }
+}
+
+export default App;
+```
 
 ## Provide the Login and Logout Buttons
 
-In the relevant location in your application, you want to provide `Login` and `Logout` buttons for the user. You can show/hide the correct button by using the `auth.isAuthenticated()` method. For example:
+In the relevant location in your application, you want to provide `Login` and `Logout` buttons for the user. You can show/hide the correct button by using the `auth.isAuthenticated()` method.  In this example,`props.auth` has been provided via the `withAuth` HOC.
```

Comment:
I don't think it's safe to assume they are comfortable with HOC, certainly not without some level of reference beyond an unexpanded acronym.